Ensure border on scrolling transcript doesn't get cut off

lyzadanger opened the pull request:
Contain element with height of 80vh with a flex-column layout. This will prevent the bottom of the border on the transcript from disappearing.

Oh! It does occur to me to ask what the purpose is of constraining that element to 80vh, but we can tackle that separately!

Before:

<img width="875" alt="image" src="https://github.com/hypothesis/via/assets/439947/ea59eceb-e05c-42fc-aff8-615457552150">


After:

<img width="881" alt="image" src="https://github.com/hypothesis/via/assets/439947/fb9f83af-6168-458b-bcb3-c604759931ce">


Also, just for style elimination: default border color should be `grey-3` without having to set; I removed the border-color styles there because they were right in front of me :).

lyzadanger committed:
Ensure border on scrolling transcript doesn't get cut off

Contain element with height of 80vh with a flex-col layout.

robertknight commented:
> It does occur to me to ask what the purpose is of constraining that element to 80vh, but we can tackle that separately!

I think it makes sense to size the transcript to some fraction of the viewport size, maybe with min/max limits. This balances ensuring the whole scrollable container fits in the viewport, and taking advantage of however tall the screen is. The 80% value is an arbitrary choice.

robertknight committed:
Increase transcript + controls height from 80vh to 90vh

Allow the user to see more of the transcript at once by using more of
the available vertical space.

robertknight reviewed (approved):
LGTM. Following this fix I increased the transcript height from 80vh to 90vh so that overall it is roughly the same height as before.
